bedrock/foundation/templates/foundation/trademarks/policy.html

```diff
@@ -158,30 +158,24 @@ <h4>{{ _('You Must NOT:') }}</h4>
 
   <h4>{{ _('You May:') }}</h4>
 
-  <ul>
+  <ul class="mzp-u-list-styled">
 
     <li>{{ _('State in words (not using logos or images) that your product “works with” or “is compatible” with certain Mozilla software products, if that is true. ') }}</li>
 
     <li>{{ _('State in words (not using logos or images) that your project is based on Mozilla open source technology, if that is true, as long as you also include a statement that your project is not officially associated with Mozilla or its products. ') }}
-
-      <p>{{ _('For instance, you may state that your project:') }}</p>
-
+      <br>
+      <br>
+      <p class="no-line-spaceing">{{ _('For instance, you may state that your project:') }}</p>
```

Comment:
Typo: this should probably be `no-line-spacing`?

Nit pick: either way this is a presentational class name, which is something we generally try and avoid where possible. Perhaps a more semantically appropriate name could be used here instead? Perhaps something like `class="lead-in"`?

Reply:
I defer to you on this as long as the the spacing stays the same on the page.